packages/build/src/scripts/generateDefaultTypes.js

```diff
@@ -27,8 +27,13 @@ const defaultPackages = [
   '@lowdefy/blocks-loaders',
   // '@lowdefy/blocks-markdown',
   '@lowdefy/connection-axios-http',
+  '@lowdefy/operators-change-case',
+  // '@lowdefy/operators-diff',
   '@lowdefy/operators-js',
+  // '@lowdefy/operators-mql',
```

Comment:
Are these default operators correct? Maybe we just need to go over all the defaults.